### ISSUE TITLE
参数

### ISSUE THREAD

**Issue by zs-joker** [USTC-PMLAB/U_DICNet#1]
Epoch 85: Current EPE = 0.0000, Best EPE so far = -1.0000
world_size=1     rank=0
Parsed arguments:
arch: U_DICNet, train_dataset_root: autodl-fs/data/train_data, test_dataset_root: autodl-fs/data/test_data, pretrained: None, solver: sgd, workers: 1, batch_size: 16, milestones: [40, 80, 120, 160, 200, 240], epochs: 300, beta: 0.999, start_epoch: 0, weight_decay: 0.0004, bias_decay: 0, lr: 1e-05, momentum: 0.9, multiscale_weights: [0.08, 0.02, 0.02, 0.05, 0.24]
———————————————————————————————————————————————————
Epoch 86: Training Loss = 0.0000, Training Flow EPE = 0.0000
world_size=1     rank=0
Parsed arguments:
arch: U_DICNet, train_dataset_root: autodl-fs/data/train_data, test_dataset_root: autodl-fs/data/test_data, pretrained: None, solver: sgd, workers: 1, batch_size: 16, milestones: [40, 80, 120, 160, 200, 240], epochs: 300, beta: 0.999, start_epoch: 0, weight_decay: 0.0004, bias_decay: 0, lr: 1e-05, momentum: 0.9, multiscale_weights: [0.08, 0.02, 0.02, 0.05, 0.24]

每轮训练完参数都一样 不更新 是哪里出了问题吗
